qa-niaid.planx-pla.net/manifest.json

```diff
@@ -18,7 +18,7 @@
     "revproxy": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/nginx:2022.05",
     "sheepdog": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/sheepdog:2022.05",
     "sower": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/sower:2022.05",
-    "portal": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/data-portal:feat_studyViewerIdpFlag",
+    "portal": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/data-portal:feat_userLoginIDPRedirect",
```

Comment:
I'm just curious on why to make this a change in master. If this switch is temporary, don't we just `ssh` into the pod and change it?

Reply:
people need to review the change and can take a few days, rather then having to remote in and change it every day i just change it in master